src/components/commons/button/BarButton.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import classNames from 'classnames';
+import Image from 'next/image';
+
+type BarButtonProps = {
+  text?: string;
+  size?: 'sm' | 'lg';
+  weight?: 'string';
+  onClick?: () => void;
+};
+
+export default function BarButton({
+  text,
+  size = 'lg',
+  weight,
+  onClick,
+}: BarButtonProps) {
+  const className = classNames(
+    'w-full flex justify-center items-center text-black-800 border border-gray-200 gap-8 font-bold  hover:border-gray-400 ',
+    {
+      'py-24 rounded-8 text-18': size === 'lg' && text,
+      'py-9 rounded-6': size === 'lg' && !text,
+      'py-20 text-16 rounded-8': size === 'sm' && text,
+      'py-6 rounded-6': size === 'sm' && !text,
+    },
+  );
+  return (
+    <button type="button" className={`${className}`} onClick={onClick}>
```

Comment:
템플릿 리터럴 사용안하셔도 될 것 같아요!

```suggestion
    <button type="button" className={className} onClick={onClick}>
```